doc/manual/merging.md

```diff
@@ -17,8 +17,8 @@ operator.
 Merge is a **symmetric** operation (or, pedantically, commutative). In practice,
 this means that order doesn't matter, and `left & right` is the same thing as
 `right & left`. When the operands need to be distinguished, as we will see for
-default values for example, the idea is to use metadata to do so (annotations),
-rather than relying on the left or right position.
+default values for example, the idea is to use special annotations (metadata)
+instead of relying on the side of the operands.
```

Comment:
```suggestion
default values for example, the idea is to use special annotations (metadata)
instead of relying on the order of the operands.
```
I feel, "order" reads more naturally here.